python/cudf/cudf/tests/test_categorical.py

```diff
@@ -62,8 +62,8 @@ def test_categorical_integer():
     string = str(sr)
     expect_str = """
 0 a
-1
-2
+1 NaN
```

Comment:
Should these be `null` instead of `NaN`?

Reply:
Pandas supports NaN in categorical dataframes, which I'm supporting here as a special case.

Reply:
Previously our own formatter did Null formatting, outputting nothing. Now the Categorical Series is cast `to_pandas()`, which works as expected. I could force our Categorical output to produce `null` like `null` in other column types, but it seemed like matching Pandas was better in this case.

Reply:
We should use `null` here as Pandas uses a code of `-1` to representing missing data in categoricals and then prints `NaN` whereas we use a null bitmask.

Reply:
👍

Reply:
I might not be able to get this in before burndown expires. It seems there's an issue with string columns properly being copied with null_masks. I'm looking into it as a necessary fix.

Reply:
👍 we can always do a followup PR.

Reply:
Thanks, lets do that.